input/definitions/Resource/ValueSet-negation-reason-example.json

```diff
@@ -0,0 +1,34 @@
+{
+  "resourceType": "ValueSet",
+  "id": "negation-reason-example",
+  "url": "http://hl7.org/fhir/ValueSet/negation-reason-example",
+  "meta": {
+    "lastUpdated": "2024-07-01T07:07:38.434+11:00",
+    "profile": [
+      "http://hl7.org/fhir/StructureDefinition/shareablevalueset"
+      ]
+  },
+  "extension": [{
+    "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-wg",
+    "valueCode": "fhir"
+  }],
+  "identifier": [
+    {
+      "system": "urn:ietf:rfc:3986",
+      "value": "urn:oid:2.16.840.1.113883.4.642.2.575"
+    }
+  ],
+  "name": "NegationReasonExample",
+  "title": "Negation Reason Example",
+  "status": "active",
+  "experimental": false,
+  "date": "2024-07-01",
+  "description": "Codes for the reason the event did not occur or was not performed.",
+  "compose": {
+    "include": [
+      {
+        "system": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1021.56"
```

Comment:
Don't reference the VSAC value set here, include the first 3 of the specific SNOMED codes in the expansion of the QICore value set:

https://hl7.org/fhir/us/qicore/ValueSet-qicore-negation-reason.html#expansion